### DISCUSSION TITLE
Installing eth-brownie with Python 3.11.0 (possible issue you could face)

### DISCUSSION THREAD

**Discussion by jnealew** [smartcontractkit/full-blockchain-solidity-course-py#1794] [Q&A] [answered]
I ran into a unique issue. When attempting to install eth-brownie there were multiple issues that kept occurring. Literally nothing fixed it.
Here is the way I solved the issue of ethics-brownie not installing. I did not keep the details of the errors, though if you are out of options, this is a possible last alternative for you.
Turns out Python 3.11.0 creates a few issues when installing eth-brownie. It is an issue with the version of python3 you are running. I was able to use a older version (Python 3.9.10) and it worked:
**Note:** How I ran the older version of Python was what you might want to do differently. I wiped my Mac off all settings and data, then the first program I installed was Python 3.9.10. I was in need of a refresh. 
Change the version of python running, then run:
`python3 -m pip install --user pipx`
`python3 -m pipx ensurepath  `
`exit`
Restart Terminal
`pipx install eth-brownie`
That should install fine.

**Comment by jnealew** [comment, marked as answer]
Answer in Question

**Reply by cromewar** [reply]
Nice catch! keep learning.
